### PR TITLE
Remove configurable timeout range

### PR DESCRIPTION
**Pull request by netcriptus**
Solves https://github.com/raiden-network/raiden-contracts/issues/1550

### CODE REVIEW

File: raiden_contracts/data/source/services/MonitoringService.sol [hunk 1]
```diff
@@ -256,7 +256,7 @@ contract MonitoringService is Utils {
 
         // We don't actually know when the channel has been closed. So we'll
         // make a guess so that assumed_close_block >= real_close_block.
-        uint256 assumed_settle_timeout = token_network.settlement_timeout_min();
+        uint256 assumed_settle_timeout = token_network.settle_timeout();
```
Comment (karlb) [inline raiden_contracts/data/source/services/MonitoringService.sol:259]: Nice, this even increases accuracy! We do know the exact settle time now, don't we? If that's the case, the comment and variable name should be updated to reflect that.

**Review by karlb — APPROVED** [review]
Good to have this in! One thing less to worry about, some gas savings and nearly 500 lines less.

If we wanted to save even more gas, we could set the settle timeout as a constant in the contracts, but that would require recompilation of the contracts when changing the value. Probably not worth the effort for this case along, but it might be interesting to keep in mind if we spot more cases like it.

I would have preferred multiple small commits:
* Actual code changes
* Contract recompilation and gas measurements (could also be split in two)
* Deployment on testnets
* Deployment on mainnet

File: raiden_contracts/data/source/services/MonitoringService.sol (hunk 1 above)
Comment (netcriptus) [reply raiden_contracts/data/source/services/MonitoringService.sol]: Good point, this went unnoticed ✅

File: raiden_contracts/data/deployment_goerli.json [hunk 2]
```diff
@@ -16,8 +16,7 @@
       "gas_cost": 4579912,
       "constructor_arguments": [
         "0xDEF219a11e3d7485075704Ec9ade6eF994c0b5B7",
-        20,
-        555428,
+        200,
```
Comment (palango) [inline raiden_contracts/data/deployment_goerli.json:19]: Why did you change these? These files are written when depolying the contracts, and shouldn't be changed by hand.

Comment (netcriptus) [reply raiden_contracts/data/deployment_goerli.json:19]: I couldn't find where these values were being set, I assumed they should be manually altered in the deployment files.

Comment (palango) [reply raiden_contracts/data/deployment_goerli.json:19]: No worries, we can sit together and deploy the contracts on goerli when we're both in the office. Just to see how the process looks like and when those files are created.
